modules/test/playwright/pages/document-library-web/DocumentLibraryPage.ts

```diff
@@ -139,4 +146,8 @@ export class DocumentLibraryPage {
 			.and(this.page.locator('[aria-haspopup]'))
 			.click();
 	}
+	async orderBy(name: string) {
```

Comment:
Use [literal types](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types) to avoid constants such as `documentLibraryPage.ORDER_MENU_ITEM`

In this case:

```ts
async orderBy(name: 'Descending' | 'Modified Date' ) {
```